Automattic/jetpack
# VideoPress: TScriptify select video control of chapters block

On this PR:
* Rewrite the select video control using TS
* Keep Typifying the code 
* Teak de Details panel

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: TScriptify select video control of chapters block


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Testing instructions:
* Enable beta extensions
* Go to block editor
* Create a new post
* Insert a VideoPress video chapter  block
* Confirm the chapters block shows the no videos message
<img width="603" alt="Screen Shot 2022-12-11 at 16 26 03" src="https://user-images.githubusercontent.com/77539/206915784-86e1077b-7a9a-4016-ad69-dd26d07a72de.png">
* Add a new VideoPress video block
* Confirm the Video chapters block recognizes the video block and links it.
* Confirm that when there is only one video, the select control doesn't appear.

Video block | Video Chapters block
------|------
<img width="922" alt="Screen Shot 2022-12-11 at 16 26 46" src="https://user-images.githubusercontent.com/77539/206915978-2b3d9c1b-acd5-435e-ac6c-bc435f5ea7bc.png"> | <img width="920" alt="Screen Shot 2022-12-11 at 16 27 04" src="https://user-images.githubusercontent.com/77539/206915964-26fc4093-a1d3-4469-806e-77b5ee6afc02.png">

* Add more VideoPress videos
* Confirm the Video chapters block shows the select control
<img width="457" alt="Screen Shot 2022-12-11 at 16 32 50" src="https://user-images.githubusercontent.com/77539/206916212-dcea0da7-c95e-4f05-8ab5-e62b3dd5af7b.png">

### Known issues

* If you save the post and then hard-refresh, the Video chapters block loses the link with the Video block. https://github.com/Automattic/jetpack/issues/27872
